src/hazmat/sieve.rs

```diff
@@ -9,25 +9,26 @@ use rand_core::CryptoRngCore;
 
 use crate::hazmat::precomputed::{SmallPrime, RECIPROCALS, SMALL_PRIMES};
 
-/// Returns a random odd integer with given bit length
-/// (that is, with both `0` and `bit_length-1` bits set).
+/// Returns a random odd integer with given bit length (that is, with both `0` and `bit_length-1`
```

Comment:
This is not going to affect merging this PR, but I just want to note that the original formatting was intentional - it's easier to read text that is 50-70 characters wide, so I try to add line breaks at meaningful points in the sentences.